layouts/partials/head.html

```diff
@@ -95,11 +92,275 @@
+      font-size: 0.85rem;
+      color: #999;
+    }
+
+    /* Footer metadata area within the blog card */
+    .blog-card-meta {
+      margin-top: 0.5rem;
+      font-size: 0.9rem;
+    }
+
+    .blog-single-container {
+      max-width: 800px;
+      margin: 0 auto;
+      text-align: left;
+      font-size: 1.2rem;
+    }
+
+    .tag-list {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 0.5rem;
+      list-style: none;
+      padding: 0rem;
+    }
+
+    .tag-card {
+      background-color: #e0e0e0;
+      border-radius: 4px;
+      padding: 0.5rem 1rem;
+    }
+
+    .tag-card a {
+      text-decoration: none;
+      color: #333;
+      font-weight: 500;
+    }
+
+    .tag-card a:hover,
+    .tag-card a:focus {
+      color: #0052cc;
+    }
+
+    .blog-meta {
+      font-size: 1rem;
+      color: #2f2d2d;
+      margin-top: 0.5rem;
+      margin-bottom: 1.5rem;
+    }
+
+    .blog-meta span {
+      margin-right: 0.5rem;
+    }
+
+    .blog-canonical a {
+      color: #333;
+      /* Neutral color */
+      text-decoration: underline;
+      /* Always underline */
+    }
+
+    .blog-canonical a:visited {
+      color: #333;
+      /* Remove purple color once clicked */
+    }
+
+    .blog-canonical a:hover,
+    .blog-canonical a:focus {
+      color: #0052cc;
+      /* Blue on hover/focus */
+    }
+
+    .toc-card {
+      background-color: #f5f5f5;
+      border-radius: 8px;
+      padding: 0.5rem;
+      margin-bottom: 1.2rem;
+    }
+
+    .toc-title {
+      font-weight: 700;
+      font-size: 1.1rem;
+      color: #333;
+    }
+
+    .toc-content {
+      margin-top: 0.5rem;
+    }
+
+    .toc-list {
+      list-style: none;
+      padding: 0;
+    }
+
+    .toc-item a {
+      color: #333;
+      /* Always black */
+      text-decoration: none;
+      /* No underline */
+    }
+
+    .toc-item a:visited {
+      color: #333;
+      /* Remove purple color once clicked */
+    }
+
+    .toc-item a:hover,
+    .toc-item a:focus {
+      color: #0052cc;
+      /* Blue on hover/focus */
+      text-decoration: none;
+      /* No underline */
+    }
+
+    .footer-nav-card {
+      display: flex;
+      justify-content: space-between;
+      background-color: #f5f5f5;
+      border-radius: 8px;
+      padding: 0.5rem;
+      margin-top: 1rem;
+    }
+
+    .footer-nav-link {
+      flex: 1;
+      text-align: center;
+      padding: 0.5rem;
+      color: #333;
+      text-decoration: none;
+      font-weight: 500;
+    }
+
+    .footer-nav-link:hover,
+    .footer-nav-link:focus {
+      background-color: #dcdcdc;
+      color: #333;
+      text-decoration: none;
+    }
+
+    .footer-nav-prev {
+      border-right: 1px solid #ccc;
+    }
+
+    .footer-nav-next {
+      border-left: 1px solid #ccc;
+    }
+
+    .footer-nav-text {
+      display: block;
+      font-size: 0.9rem;
+      margin-bottom: 0.5rem;
+    }
+
+    .footer-nav-title {
+      display: block;
+      font-size: 1rem;
+      font-weight: 700;
```

Comment:
Can we try to reduce this a little bit? I'd like to have as few overrides to Vanilla as possible, so that it's easy to keep up to date as the design system evolves? 

This feels like an excessive amount of change? I could be wrong, but I'd like to use standard components as much as we can.

Reply:
It leads to inconsistencies like the following:

![image](https://github.com/user-attachments/assets/9a2452e1-be49-4a70-abe0-b37566a311c2)

![image](https://github.com/user-attachments/assets/48970f32-94f9-44b9-b169-240459585beb)

Note the alignment difference highlighted in red?

Reply:
Hello @jnsgruk, thanks a lot for the feedback. I'll take up this PR from here, and will migrate the CSS to the vanilla framework.